prover/crypto/keccak/keccakf.go

```diff
@@ -5,7 +5,6 @@ package keccak
 const Dim = 5           // size of the keccak state matrix
 const NumRound int = 24 // number of rounds in the keccak permutation
 
-// LR collects the triangular numbers for the \pi and \rho step
```

Comment:
Are there performances gains that are registered from these changes? The implementation is a toy implementation that is designed to make it, easier to test and debug the keccak arithmetization. But AFAICT, this had not been flagged as a performance critical piece of code (this is still used in production to produce the witness).

Reply:
If the gains are negligible, I would rather ditch those changes

Reply:
Got it, let me perform an additional benches to check the performance of this feature separately. I will share the results after